### PR TITLE
chore: adds theme-ui example project

### PR DESCRIPTION
**Pull request by guatedude2**
This PR provides a Theme-UI Remix example with support for SSR.

Closes: https://github.com/remix-run/remix/discussions/3636

- [x] Docs
- [ ] Tests

Testing Strategy:

Tested example locally

### CODE REVIEW

**Comment by guatedude2** [comment]
https://github.com/system-ui/theme-ui/discussions/2231

File: examples/theme-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+*This implementation was based off [Saas-UI Remix guide](https://www.saas-ui.dev/docs/core/installation/remix-guide).*
+
+### Theme-UI Related files
+
+```
+- app/
+  - styles/
    - context.tsx
    - createEmotionCache.tsx
```
Comment (kentcdodds) [inline examples/theme-ui/README.md:15]: image

Do these newlines show up funny to anyone else?

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks! This looks good to me. Only thing is the funny newline character in the README.

File: examples/theme-ui/README.md (hunk 1 above)
Comment (kentcdodds) [reply examples/theme-ui/README.md:15]: Looks like it renders fine so I'll just merge. Thanks!

**Review by kentcdodds — APPROVED** [review]
We're good 👍

File: examples/theme-ui/README.md (hunk 1 above)
Comment (guatedude2) [reply examples/theme-ui/README.md:15]: Odd, I didn't see those. Thanks for approving :)

**Pull request merged**